diff -urN harfbuzz-4.1.0.orig/configure harfbuzz-4.1.0/configure
--- harfbuzz-4.1.0.orig/configure	2022-03-23 06:22:41.000000000 +0100
+++ harfbuzz-4.1.0/configure	2022-03-26 16:11:38.350750097 +0100
@@ -19330,7 +19330,7 @@
 
 fi
 if test -n "$PKG_CONFIG"; then
-	_pkg_min_version=0.28
+	_pkg_min_version=0.27
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking pkg-config is at least version $_pkg_min_version" >&5
 $as_echo_n "checking pkg-config is at least version $_pkg_min_version... " >&6; }
 	if $PKG_CONFIG --atleast-pkgconfig-version $_pkg_min_version; then
